baked_in.go

```diff
@@ -1370,6 +1371,23 @@ func isURL(fl FieldLevel) bool {
 	panic(fmt.Sprintf("Bad field type %T", field.Interface()))
 }
 
+// isHttpURL is the validation function for validating if the current field's value is a valid HTTP(s) URL.
+func isHttpURL(fl FieldLevel) bool {
+	if !isURL(fl) {
+		return false
+	}
+
+	field := fl.Field()
+	switch field.Kind() {
+	case reflect.String:
```

Comment:
It'd be nice to see support for `*string` as thats the most used cases in most of the generators.

Reply:
strings should be dereferenced by the time it reaches this point in the code.